### ISSUE TITLE
CDI 2.0: create all auto features 

### ISSUE THREAD

**Issue by Emily-Jiang** [OpenLiberty/open-liberty#250]
Create auto features with ejb, client, jms, jndi, jsf, security, transaction, web

**Comment by tevans78** [comment]
are you sure that jms and transactions should be included in this task?

**Comment by tevans78** [comment]
Before PR #579 was integrated, @Channyboy had some problems with jms and ejb. Need to go back and double check these...

`com.ibm.websphere.appserver.wasJmsClient-2.0'`  was the feature i asked about yesterday
Error complained about `com.ibm.ws.messaging.jms.2.0.cdi` not resolving javax.enterprise.blabla and you advised to stick:
```    javax.enterprise.context.*; version="[1.1.0,3.0.0)", \
    javax.enterprise.event.*; version="[1.1.0,3.0.0)", \
    javax.enterprise.inject.*; version="[1.1.0,3.0.0)", \
    javax.enterprise.util.*; version="[1.1.0,3.0.0)", \
    javax.decorator; version="[1.1.0,3.0.0)", \
    *
```
into the bnd.bnd of that project

Similar issue with `com.ibm.websphere.appserver.ejbRemoteClient-1.0` feature and complained about the `com.ibm.ws.cdi.ejb` module not resolving.
I tried similar tactic to import those packages, it works for that, but then starts complaining about org.jboss.weld, trying similar work around with those packages, but then eventually complained about `org.jboss.weld.ejb`


ALSO, as an aside, there was an issue with  ` com.ibm.websphere.appserver.jndiClient-1.0` because it was still explicitly depending on CDI1.2.

**Comment by tevans78** [comment]
This issue is more or less covered by PR #579 and #659 and can be closed once #659 is merged.

**Comment by tevans78** [comment]
There are a few more client related issues as noted above but I suspect that they will also be resolved by #659. If not they we will open a new issue.

**Comment by tevans78** [comment]
I was going to create a cdi2.0-batch1.0 auto feature but andy just changed the existing cdi1.2 feature to also allow cdi2.0. That'll do for now. This issue can be closed.
